src/App/IndexedName.h

```diff
@@ -147,6 +147,18 @@ class AppExport IndexedName {
         return result;
     }
 
+    const char * toString(std::string & s) const
```

Comment:
My intent was that this be replaced by `appendToStringBuffer` -- do we need the char * return someplace? (Sorry, this was early days of the project, documentation is not what it should be)

Reply:
Nope, I was being thorough after seeing the use in MappedName.  Missed the forest for the trees.  This is an object lesson in not letting an old semi-complete PR hang around.  I knew so little back when this was first written.